### PR TITLE
feat(models): adds knex and bookshelf config and movie object model

### PR DESCRIPTION
**Pull request by dmlittle**
What: 
- Add `Knex` and `Bookshelf` to the repo.
- Add `Movie` Bookshelf model.

Why: Sets up the base for creating the `/movies` endpoint to work with the database.

@elnaz

### CODE REVIEW

File: db/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  client: 'postgresql',
+  connection: {
+    database: Config.DB_NAME,
+    host: Config.DB_HOST,
+    password: Config.DB_PASSWORD,
+    port: Config.DB_PORT,
+    user: Config.DB_USER
+  }
+};
+
```
Comment (elnaz) [inline db/index.js]: nit: extra line

File: db/migrations/20160713150839_create_movies.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    table.increments('id').primary();
+    table.text('title').notNullable();
+    table.integer('release_year');
+  });
+};
+
+exports.down = function (Knex, Promise) {
+  return Knex.schema.dropTable('movies');
+};
+
```
Comment (elnaz) [inline db/migrations/20160713150839_create_movies.js]: nit: extra line

**Comment by elnaz** [comment]
LGTM

**Comment by robinjoseph08** [comment]
LGTM

**Pull request merged**